CONTRIBUTING.rst

```diff
@@ -111,6 +111,26 @@ Please note that there is a minimum review period of 24 hours for any patch. The
 
 All subsequent changes made to patches must be force-pushed to the PR branch before merging it into the main branch.
 
+Code conventions
+----------------
+
+It is important to have consistency across the codebase. This won't necessarily make your code work better, but it might help make it more understandable, time-consuming and less irritating to go through when doing a code review of your changes.
+
+In general we are trying to be as close as possible to `PEP8 <https://www.python.org/dev/peps/pep-0008/>`_ but also extending or modifying minor rules when it looked more readable to us. See list of the conventions below:
```

Comment:
`but also extending or modifying minor rules when it looked more readable to us.`
I suggest:
`but also extending or modifying minor PEP8 rules when it seems suitable in the context of our project.`

Reply:
+1, the suggestion makes it more neutral

Reply:
Fixed.